### PR TITLE
dataProfile examples

### PR DESCRIPTION
**Pull request by ldecicco-USGS**
Getting read for CRAN:
1. Add some hard coded column types
2. Add a "guess_max" for how far down R should guess for picking unknown types
3. Add one example in  readWQPsummary help file
4. Add a bunch of dataProfile examples.

### CODE REVIEW

File: man/readWQPsummary.Rd [hunk 1]
```diff
@@ -68,9 +68,10 @@ dane_county_data <- readWQPsummary(countycode = "US:55:025",
```
Comment (lstanish-usgs) [inline man/readWQPsummary.Rd:68]: Can you add a descriptor for these examples? e.g 'Obtain a summary table of all stream water quality data for the state of New Jersey from the past 5 years'

File: man/readWQPdata.Rd [hunk 2]
```diff
@@ -146,6 +146,54 @@ dailyLexingtonVA <- readWQPdata(statecode = "Virginia",
                                 countycode="Lexington", 
                                 parameterCd = "00010") 
                                 
+# Data profiles: "Organization Data"
```
[the remark this answers is not in the document]
Comment (ldecicco-USGS) [reply man/readWQPdata.Rd:149]: I'm unclear if there are dataProfiles that exists in multiple services. Do you happen to know?

[the remark this answers is not in the document]
Comment (ldecicco-USGS) [reply man/readWQPdata.Rd:149]: If that question doesn't make sense...what I mean is... I used the exact examples that are listed here:

image

Do you know if those are the exclusive list of possibilities...or could you ask for Results with a dataProfile of "activityAll" for example (the example in the UI suggest activityAll would go with the Activity service....but 🤷‍♀️

File: R/readNWISdata.r [hunk 3]
```diff
@@ -276,21 +276,22 @@ countyCdLookup <- function(state, county, outputType = "id"){
     stop("No county code provided")
   }
   
+  if(length(state) > 1){
+    stop("Only one state allowed in countyCdLookup.")
+  }
```
[the remark this answers is not in the document]
Comment (ldecicco-USGS) [reply R/readNWISdata.r:281]: for example, in readWQP or readNWIS, how are multiple counties handled, especially over multiple states?

File: R/readNWISqw.r [hunk 4]
```diff
@@ -83,8 +83,7 @@ readNWISqw <- function (siteNumbers,parameterCd,startDate="",endDate="",
                         expanded=TRUE,reshape=FALSE,tz="UTC"){  
   
   .Deprecated(new = "readWQPqw", package = "dataRetrieval", 
-              msg = "NWIS qw web services are being retired. Please see the vignette 
-'Changes to NWIS QW services' for more information.")
+              msg = "NWIS qw web services are being retired. Please see vignette('qwdata_changes', package = 'dataRetrieval') for more information.")
```
Comment (ldecicco-USGS) [inline R/readNWISqw.r:86]: or:
https://cran.r-project.org/web/packages/dataRetrieval/vignettes/qwdata_changes.html
?

File: R/readWQPdots.R [hunk 5]
```diff
@@ -23,7 +23,20 @@ readWQPdots <- function(...){
...
+      matchReturn$service <- NULL
+    } else if(profile %in% c("resultPhysChem",
+                             "biological",
+                             "narrowResult")){
+      service <- "Result"
+      matchReturn$service <- NULL
+    }
+  }
+  
+  match.arg(service, c("Result", "Station", "Activity", "Organization",
```
Comment (ldecicco-USGS) [inline R/readWQPdots.R:39]: Double check

File: R/tabbedDataRetrievals.R [hunk 6]
```diff
@@ -127,4 +129,38 @@ NULL
 #' @keywords USGS countyCd
 #' @examples
 #' head(countyCd)
-NULL
\ No newline at end of file
+NULL
+
+
```
Comment (ldecicco-USGS) [inline R/tabbedDataRetrievals.R:134]: Add a comment to describe what's happening.

**Pull request merged**